.gitleaks.toml

```diff
@@ -57,8 +60,8 @@ title = "gitleaks config"
     regex = '''(?i)(api_key|apikey|secret)(.{0,20})?['|"][0-9a-zA-Z]{16,45}['|"]'''
     tags = ["key", "API", "generic"]
     [rules.allowlist]
-        paths = ['''^examples\/.*''']
-        description = "ignore examples"
+	regexes = ['''(3e34709d01ea89032asdebfe5a74518|12345670)''']
```

Comment:
Out of curiosity, where did you get it? 😄  I mean the tokens

Reply:
We use `3e34709d01ea89032asdebfe5a74518` in [docs, and tests](https://github.com/RasaHQ/rasa/search?q=3e34709d01ea89032asdebfe5a74518). 
Same for `123456701ea89032asdebfe5a74518` 👉 it starts with `12345670` so we can just match this part.